en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md

```diff
@@ -0,0 +1,318 @@
+# FHIR Connector Example
+
+Given below is a sample scenario that demonstrates how to work with resources in [FHIR](https://www.hl7.org/fhir/overview.html)(Fast Healthcare Interoperability Resources). FHIR specification is a standard for exchanging health care information electronically.The WSO2 EI FHIR connector uses the FHIR RESTFul API to interact with FHIR.
+
+## What you'll build
+
+Given below is a sample API that illustrates how you can connect to a FHIR with the `init` operation and then use the `create`,`readResource`, `readSpecificResourceById`,`delete` and `update` operation to send messages to the FHIR server.
+
+* `<fhir.init>` : It will initialize the FHIR operations.
+* `<fhir.create>` : To creates a new resource in a server-assigned location, use `fhir.create` operation.
+* `<fhir.readResource>` : To accesses the current contents of a resource, use `fhir.readResource` operation.
+* `<fhir.readSpecificResourceById>` : To accesses the specific current contents of a resource, use `fhir.readSpecificResourceById` operation.
+* `<fhir.update>` : To creates a new current version for an existing resource or creates an initial version if no resource already exists for the given id, use `fhir.update` operation.
+* `<fhir.update>` : To removes an existing resource, use `fhir.delete`operation.
+
```

Comment:
These are implementation detail. Shall we talk about it like below in a `scenario mindset`. 

Given below is a sample API that illustrates how you can connect to a FHIR server and invoke operations. It exposes FHIR functionalities as a restful service. Users can invoke the API with HTTP/HTTPs with required information. 

API has the context `/resources `. It has four resources  

1. `/create` :  creates a new patient at FHIR server
2. `/read` :  retrieve information about the patient from FHIR server 
3. `/readSpecificResourceById`: <map this to the scenario is it read patient by Id? Which ID?> 
4. `/delete `: remove added patient information from FHIR server.

Reply:
Updated.